### PR TITLE
bz1869637 updated Azure installation tables

### PR DESCRIPTION
**Pull request by sheriff-rh**
Fixes bug 1869637: https://bugzilla.redhat.com/show_bug.cgi?id=1869637

Removed the `machines.platform.azure.type` entry and corrected the disk size for the `machines.platform.azure.osDisk.diskSizeGB` parameter. 

@xltian please take a look.

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -499,16 +499,9 @@ ifdef::azure[]
...
-|The Azure VM instance type.
-|VMs that use Windows or Linux as the operating system. See the
-link:https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908[Guest operating systems supported on Azure Stack]
-in the Azure documentation.
-
 |`machines.platform.azure.osDisk.diskSizeGB`
 |The Azure disk size for the VM.
-|Integer that represents the size of the disk in GB, for example `512`. The
-minimum supported disk size is `120`.
+|Integer that represents the size of the disk in GB. The minimum supported disk size is 1TB.
```
Comment (bergerhoffer) [inline modules/installation-configuration-parameters.adoc:504]: Just a quick question whether 1TB should be written in terms of GB, if that's what needs to go into this field?

**Review by bergerhoffer — COMMENTED** [review]
Just a quick question, but LGTM!

File: modules/installation-configuration-parameters.adoc (hunk 1 above)
Comment (sheriff-rh) [reply modules/installation-configuration-parameters.adoc:504]: thanks @bergerhoffer  i just saw in [this PR](https://github.com/openshift/installer/pull/2367)  that it needs to be `1024`. i'll fix it! thanks!

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -499,16 +499,9 @@ ifdef::azure[]
 |====
 |Parameter|Description|Values
 
-|`machines.platform.azure.type`
-|The Azure VM instance type.
-|VMs that use Windows or Linux as the operating system. See the
-link:https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908[Guest operating systems supported on Azure Stack]
-in the Azure documentation.
-
 |`machines.platform.azure.osDisk.diskSizeGB`
```
Comment (jianlinliu) [inline modules/installation-configuration-parameters.adoc:502]: The PR https://github.com/openshift/installer/pull/2367 was bumping up controlPlane's disk size. And I even do not know install-config.yaml has `machines` field. I think here should be `controlPlane.platform.azure.osDisk.diskSizeGB`

**Comment by bergerhoffer** [comment]
@vikram-redhat This PR is going stale, just wanted to see if you wanted to assign the BZ out to someone else to see if they can push it forward.

**Comment by vikram-redhat** [comment]
@GroceryBoyJr PTAL. I have assigned you the BZ.

**Comment by GroceryBoyJr** [comment]
This BZ passed on to James Brigman (GroceryBoyJr) after sherrif-rh departure. GBJr changed `machines.platform.azure.osDisk.diskSizeGB` to 'controlPlaneplatform.azure.osDisk.diskSizeGB` per jianlinliu feedback on Oct. 26, 2020.

**Comment by GroceryBoyJr** [comment]
in modules/installation-configuration-parameters.adoc "machines.platform.azure.osDisk.diskSizeGB" changed to "controlPlane.platform.azure.osDisk.diskSizeGB"

**Comment by GroceryBoyJr** [comment]
This update satisfies  https://bugzilla.redhat.com/show_bug.cgi?id=1869637
document URL is:
https://docs.openshift.com/container-platform/4.3/installing/installing_azure/installing-azure-customizations.html#installation-azure-config-yaml_installing-azure-customizations

**Pull request closed without merging**